spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/reload/ConfigReloadUtil.java

```diff
@@ -84,14 +85,24 @@ public static <S extends PropertySource<?>> List<S> findPropertySources(Class<S>
 			else if (sourceClass.isInstance(source)) {
 				managedSources.add(sourceClass.cast(source));
 			}
+			else if (source instanceof MountConfigMapPropertySource mountConfigMapPropertySource) {
```

Comment:
the other 1/2 of the fix. Also a bit down in the code also, for the bootstrap case